docs/sources/k6/next/javascript-api/jslib/aws/EventBridgeClient/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: 'EventBridgeClient'
-head_title: 'EventBridgeClient'
 description: 'EventBridgeClient allows interacting with AWS EventBridge service'
 description: 'EventBridgeClient class allows sending custom events to Amazon EventBridge so that they can be matched to rules.'
```

Comment:
I see that you dropped them where they do *not* differ, but maybe we cacn choose on `description` in this case as well